src/wechaty_puppet_hostie/puppet.py

```diff
@@ -194,7 +194,14 @@ async def message_image(self, message_id: str, image_type: ImageType
         if response is None:
             # TODO -> need to refactor the raised error
             raise ValueError('response is invalid')
-        return FileBox.from_base64(response.filebox)
+        json_response = json.loads(response.filebox)
+        if 'base64' not in json_response:
```

Comment:
We might have other type of the filebox as well, like the type of `qrcode` and the `remoteUrl`.

Please read the FileBox docs to get more information.

See: https://github.com/huan/file-box/issues/25